app/src/main/java/com/example/music/player/view/presenter/songs/SongsPresenter.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.music.player.view.presenter.songs
+
+import com.example.music.player.model.entity.Song
+import com.example.music.player.view.presenter.BasePresenter
+
+interface SongsPresenter :
+    BasePresenter<SongsView> {
+    fun getSongs()
+    fun playSong(song: Song)
+    fun stopPlayingSong()
+    fun isPlaying():Boolean
+    fun pauseSong()
+    fun restartPausedSong()
```

Comment:
View should not ask about restart smth, bcs it is based on state. You should say Play or Pause or Stop and that's it. SongManager will apply required action (resume or play)